frontendservice/src/router/index.js

```diff
@@ -110,10 +115,16 @@ router.beforeEach((to, from, next) => {
     verified = currentUser.emailVerified;
   }
 
-  const requiresLogout = to.matched.some((record) => record.meta.requiresLogout);
+  const requiresLogout = to.matched.some(
```

Comment:
Nit. You've changed the way the original code was placed here for three out of the four consts. Doesn't look uniform now. Previous placement was fine I guess. Doesn't affect the logic in any way though.

Reply:
I have not changed it. It happens automatically if code is added in other areas of the page.

Reply:
You have some plugin installed which does that.

Reply:
I think priyanka said we can ignore these.